lib/buffer.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable require-buffer */
 'use strict';
```

Comment:
I'm pretty sure it does exclude comments, considering the amount of files with copyright hats at the very top.